kratos/gui/platformscreen.py

```diff
@@ -257,7 +261,11 @@ def seconds(self):
 
     @seconds.setter
     def seconds(self, seconds: int):
-        self._seconds = str(seconds) if seconds >= 10 else f"0{str(seconds)}"
+        if seconds == None:
```

Comment:
```suggestion
        if seconds is None:
```
https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html